dprint.json

```diff
@@ -14,7 +14,7 @@
   "excludes": ["**/target", "**/sqlx-data.json", "maker-frontend/dist", "taker-frontend/dist", "**/node_modules"],
   "plugins": [
     "https://plugins.dprint.dev/markdown-0.9.2.wasm",
-    "https://plugins.dprint.dev/rustfmt-0.4.0.exe-plugin@c6bb223ef6e5e87580177f6461a0ab0554ac9ea6b54f78ea7ae8bf63b14f5bc2",
+    "rustfmt.exe-plugin@7723878c1be78a571c37dcb7f92a0c5780fcfd714a23498260b575efb7e610f3",
```

Comment:
I guess you are changing this because of some NixOS stuff? :)

Perhaps look into [`git update-index --assume-unchanged`](https://www.git-scm.com/docs/git-update-index). That would allow you to make these changes locally and avoid accidentally committing them!

Reply:
yeh the plugin doesnt run on nixos